keylime/config.py

```diff
@@ -315,6 +315,8 @@ def valid_regex(regex):
 # measured boot addons
 MEASUREDBOOT_PCRS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15]
 MEASUREDBOOT_ML = '/sys/kernel/security/tpm0/binary_bios_measurements'
+MEASUREDBOOT_IMPORTS = get_config().get('cloud_verifier', 'measured_boot_imports', fallback='').split(',')
+MEASUREDBOOT_POLICYNAME = get_config().get('cloud_verifier', 'measured_boot_policy_name', fallback='accept-all')
```

Comment:
Right now this patch makes the measured boot policy global. Will that be too inflexible going forward? Should it be a per-machine setting set with the tenant command? I don't think it should necessarily hold up this PR, but something to look into for the future.

Reply:
@mpeters you're absolutely right... next step on it is to make the "measured_boot" policy a twin of the - about to be merged - "ima_policy" (i.e., these policy might be global or per-node).